static/js/ui.js

```diff
@@ -122,28 +122,38 @@ function message_hover(message_row) {
     current_message_hover = message_row;
 }
 
-exports.report_message = function (response, status_box, cls) {
+exports.report_message = function (response, status_box, cls, type) {
     if (cls === undefined) {
         cls = 'alert';
     }
 
-    status_box.removeClass(status_classes).addClass(cls)
+    if (type === undefined) {
+        type = ' ';
+    }
+
+    if (type === 'subscriptions-status') {
+        status_box.removeClass(status_classes).addClass(cls).children('#response')
+              .text(response).stop(true).fadeTo(0, 1);
+    } else {
+        status_box.removeClass(status_classes).addClass(cls)
               .text(response).stop(true).fadeTo(0, 1);
+    }
+
```

Comment:
I added elements to the div(#subscriptions-status)
`  <div class="alert" id="subscriptions-status">
      <span id="response"></span>
      <span id="close-subscriptions-status"><i class="icon-vector-remove"></i></span>            </div>`

And .text() replaces all the content of an element even including child elements, so without messing with the earlier code I checked whether the call is coming from subscriptons-status or not, if yes then I changed the content(..text()) of only '#response' span.

I could've done it with the existing framework but then I would've to show and hide 'remove' button every time explicitly. Do you dee a better approach here?